File: em Vídeo/POO [60 Horas]/Java POO [40 Horas]/Aula 09/Pessoa09.java
public class Pessoa09 {
    private String nome;
    private int idade;
    private String sexo;

    public Pessoa09(String nome, int idade, String sexo) {
        this.nome = nome;
        this.idade = idade;
        this.sexo = sexo;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public int getIdade() {
        return idade;
    }

    public void setIdade(int idade) {
        this.idade = idade;
    }

    public String getSexo() {
        return sexo;
    }

    public void setSexo(String sexo) {
        this.sexo = sexo;
    }

    public String toString() {
        return (""
                + "\tNome: " + this.getNome()
                + "\n\tIdade: " + this.getIdade()
                + "\n\tSexo: " + this.getSexo());
    }

    public void fazerAniver() {
        this.setIdade(this.getIdade() + 1);
    }
}

File: CursoEmVideo/POO [60 Horas]/PHP POO [40 Horas]/Aula 09/Pessoa09.php
<?php
    class Pessoa09 {
        private $nome;
        private $idade;
        private $sexo;

        function __construct($nome, $idade, $sexo) {
            $this->nome = $nome;
            $this->idade = $idade;
            $this->sexo = $sexo;
        }

        function getNome() {
            return $this->nome;
        }

        function setNome($nome) {
            $this->nome = $nome;
        }

        function getIdade() {
            return $this->idade;
        }

        function setIdade($idade) {
            $this->idade = $idade;
        }

        function getSexo() {
            return $this->sexo;
        }

        function setSexo($sexo) {
            $this->sexo = $sexo;
        }
        
        function toString() {
            return (""
                    ."\tNome: ".$this->getNome()
                    ."\n\tIdade: ".$this->getIdade()
                    ."\n\tSexo: ".$this->getSexo());
        }

        function fazerAniver() {
            $this->setIdade($this->getIdade() + 1);
        }
    }
